SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -100,6 +100,13 @@ void USpatialReceiver::LeaveCriticalSection()
 
 	for (Worker_EntityId& PendingAddEntity : PendingAddActors)
 	{
+		if (IncomingEntitiesToIgnore.Contains(PendingAddEntity)) // Prevent ReceiveActor for these
+		{
+			PendingDeleteWhenAuthoritive.Add(PendingAddEntity);
+			IncomingEntitiesToIgnore.Remove(PendingAddEntity);
```

Comment:
I think we need to do the `PendingAddComponents.RemoveAll` here also. Perhaps instead we can prevent adding ops for these entities in the first place?

Reply:
I'd also be pro just ignoring any op for one of these entities that isn't an authority delegation. And on that op we delete the entity.